edifact-edi/2.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,107 @@
+= EDIFACT EDI Connector 2.2 - Mule 4
+:page-aliases: connectors::edifact/edifact-edi-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector for EDIFACT EDI (EDIFACT Connector) enables you to convert EDIFACT messages to and from DataWeave-compatible representations using lists and maps.
+
+The  xref:edifact-edi-versions.adoc[supported EDIFACT versions] are provided.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+To use EDIFACT EDI Connector, you must have:
+
+* Purchased a MuleSoft license for Anypoint B2B
+
+* Anypoint Studio 7.0 or later
+
+[NOTE]
+
+EDIFACT Connector supports files of up to 15 MB in size, and the memory requirement is approximately 40 to 1.
+For example, a 1-MB file requires up to 40 MB of memory to process, so it's important to consider this memory requirement in conjunction with your Transactions Per Second (TPS)  needs for large X12 files.
+This is not an exact figure; the value varies according to the complexity of the mapping instructions.
+
+=== Understanding the EDI Schema Language
+
+The EDIFACT EDI Connector uses a YAML format called EDI Schema Language (ESL) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of
+
+* structures, known as messages, in EDIFACT terminology
+
+* groups, segments, composites, and elements
+
+You can customize the EDIFACT schema definitions to suit your data by defining an overlay schema. An overlay schema is a type of ESL that enables you to modify a base schema, such as an EDIFACT D.98B ORDERS schema, with your specific conventions. You don't need an overlay schema if you're using the structure defined by the standard, but most EDI exchanges modify the base definitions in an implementation convention.
```

Comment:
```suggestion
You can customize the EDIFACT schema definitions to suit your data by defining an _overlay schema_. An overlay schema is a type of ESL that enables you to modify a base schema, such as an EDIFACT D.98B ORDERS schema, with your specific conventions. You don't need an overlay schema if you're using the structure defined by the standard, but most EDI exchanges modify the base definitions in an implementation convention.
```

Reply:
Do we need to explain this further:
"but most EDI exchanges modify the base definitions in an implementation convention."